docs/source/usage/benchmarks.rst

```diff
@@ -75,6 +75,16 @@ To run:
 then the file generated are:  ../samples/8a_parallel_3Db_*
 
 
+Optional input parameter: pack
+
+Often a processor will write out a few small blocks.
+Using the example above, if the writer side uses 1024 processors, each processor will handle 16 blocks from the 32x32x16 grid.
+The selection of these 16 blocks can be from 1x1x16, to 2x2x4, etc..
+To specify which, the ``pack`` parameter can be specified, e.g., ``"pack=1 1 16"`` or ``"pack=2 2 4"``.
```

Comment:
```suggestion
These 16 blocks per processor can be packed into near-by selections of 1x1x16 blocks, or 2x2x4, etc.
The ``pack`` parameter specifies this selection, e.g., ``"pack=1 1 16"`` or ``"pack=2 2 4"``.
```

Reply:
As a comment: just to mention, this is not exactly how AMReX does this.
AMReX uses either a space-filling Z curve or a simpler "knapsack" distribution when assigning boxes to MPI ranks.